lib/IDE/CodeCompletionDiagnostics.cpp

```diff
@@ -157,31 +157,3 @@ bool swift::ide::getContextFreeCompletionDiagnostics(
   }
   return true;
 }
-
-bool swift::ide::getContextualCompletionDiagnostics(
-    ContextualNotRecommendedReason Reason, const ValueDecl *D,
-    CodeCompletionDiagnosticSeverity &Severity, llvm::raw_ostream &Out) {
-  CodeCompletionDiagnostics Diag(D->getASTContext());
-  switch (Reason) {
-  case ContextualNotRecommendedReason::InvalidAsyncContext:
-    // FIXME: Could we use 'diag::async_in_nonasync_function'?
-    return Diag.getDiagnostics(
-        Severity, Out, diag::ide_async_in_nonasync_context, D->getName());
```

Comment:
_If_ we decide not to use DiagnosticEngine, remove these unused `diag::` definitions.